src/auth/AxiosJwtAuthService.test.jsx

```diff
@@ -96,19 +81,18 @@ const mockApiEndpointPath = `${process.env.BASE_URL}/api/v1/test`;
 window.location.assign = jest.fn();
 const mockCookies = new Cookies();
 
-// This sets the mock adapter on the default instance
```

Comment:
We re-set all these variables on `beforeEach` now, since they need to be re-created in sync with each other for every test.  If I left them here, they'd fall out of sync when we re-configure/instantiate the `AxiosJwtAuthService` for each test.